Sming/third-party/.patches/esp-open-lwip.patch

```diff
@@ -24,10 +24,10 @@ index e69de29..0c5dff9 100644
 +
```

Comment:
> This file shouldn't be here?

I don't understand. Without this file LWIP will not be patched. Can you elaborate a bit what shouldn't be there?